macro/run_primary_vertexer_ITS.C

```diff
@@ -90,6 +96,7 @@ int run_primary_vertexer_ITS(const bool useGPU = false,
   outTree.Branch("ITSVertices", &verticesITS);
 
   // DEBUG
+#if defined(__VERTEXER_ITS_DEBUG)
   TNtuple tracklets("Tracklets", "Tracklets", "oX:oY:oZ:c1:c2:c3:DCAvtx:DCAz");
```

Comment:
This might be ok provided you know a priori that the output is small, but memory-based trees are in general very dangerous. 
For the future, note that there is a very convenient class (ported from Marian's aliroot version), https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Utils/include/CommonUtils/TreeStreamRedirector.h, see https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Utils/test/testTreeStream.cxx or https://github.com/AliceO2Group/AliceO2/blob/60d17b210f4346485673203cabc08d5a291384de/Detectors/GlobalTracking/src/MatchTPCITS.cxx#L1477 which allows to redirect debug info into the tree.
It allows to stream also the objects, provided they have root dictionary.

Reply:
The idea is that these debug data are going to be removed very soon, I just made it quick since this is just for the steering macro.
I'll bare in mind the interface you mentioned for future, more structured debugs.

Reply:
OK, this was just a comment